ui/src/app/richskill/list/skills-list.component.ts

```diff
@@ -236,7 +240,7 @@ export class SkillsListComponent extends QuickLinksHelper {
         icon: "collection",
         primary: true,
         callback: (action: TableActionDefinition, skill?: ApiSkillSummary) => this.handleClickAddCollection(action, skill),
-        visible: (skill?: ApiSkillSummary) => this.addToCollectionVisible(skill)
+        visible: ((skill?: ApiSkillSummary) => this.addToCollectionVisible(skill))
```

Comment:
Why additional parens?  Lines 193 and 199 don't have them.  Extra parens can sometimes imply things you didn't intend, so just checking.

Reply:
Removed redundant parens